<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="studentTab">
       <attribute name="title">
        <string>Estudiantes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Apellido</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="last_name"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Usuario</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="user"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Contraseña</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="insertStudent">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateStudent">
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteStudent">
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="findStudent_searchbar"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="findStudent">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QTableWidget" name="student_table">
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Apellido</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>NoAprobaciones</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>NoArticulos</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>NoComentarios</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>usuario</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>password</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QTextEdit" name="student_logs"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="articlesTab">
       <attribute name="title">
        <string>Articulo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Título</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="title"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Artículo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="article"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Autor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="student_id"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Autorizador</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="student_id_auth"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QLineEdit" name="findArticle_searchbar"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="findArticle">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QTableWidget" name="article_table">
             <column>
              <property name="text">
               <string>Título</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Artículo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Autor</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Autorizado</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QTextEdit" name="article_logs"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="insertArticle">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateArticle">
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteArticle">
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="commentsTab">
       <attribute name="title">
        <string>Comentarios</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Articulo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="article_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Comentario</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="comment"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Autor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="student_id_2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_8">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="insertComment">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_9">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="2" column="0" colspan="2">
            <widget class="QTextEdit" name="comment_logs"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTableWidget" name="comment_table">
             <column>
              <property name="text">
               <string>Comentario</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Artículo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Autor</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
